commit/merkleroot/rmn/controller_test.go

```diff
@@ -312,6 +316,17 @@ func (ts *testSetup) waitForReportSignatureRequestsToBeSent(
 					continue
 				}
 				assert.True(t, len(req.GetReportSignatureRequest().AttributedSignedObservations) >= minObservers)
+				prevRmnNodeIDX := uint32(0)
+				for _, ao := range req.GetReportSignatureRequest().AttributedSignedObservations {
+					assert.Greater(t, ao.SignerNodeIndex, prevRmnNodeIDX)
+					prevRmnNodeIDX = ao.SignerNodeIndex
+
+					prevSrcChain := uint64(0)
+					for _, lu := range ao.SignedObservation.Observation.FixedDestLaneUpdates {
+						assert.Greater(t, lu.LaneSource.SourceChainSelector, prevSrcChain)
+						prevSrcChain = lu.LaneSource.SourceChainSelector
+					}
+				}
```

Comment:
This asserts that the signed observations are sorted right? I think you can use https://pkg.go.dev/sort#SliceIsSorted and do `assert.True(t, sort.SliceIsSorted(...))` to make that clear?